services/models/things/src/main/java/org/eclipse/ditto/services/models/things/ThingSnapshotTaken.java

```diff
@@ -105,26 +105,26 @@ public static Builder newBuilder(final ThingId thingId,
     }
 
     /**
-     * Deserializes a {@code ThingSnapshotTaken} instance from the specified JSON object.
+     * Deserializes a {@code DittoThingSnapshotTaken} instance from the specified JSON object.
      *
      * @param jsonObject the JSON object that should be deserialized.
      * @param dittoHeaders the headers of the deserialized event.
-     * @return the deserialized ThingSnapshotTaken instance.
+     * @return the deserialized BoschThingSnapshotTaken instance.
```

Comment:
Should be `ThingSnapshotTaken` instead of `BoschThingSnapshotTaken`.

Reply:
Changed.